.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: test
+
+on:
+  schedule:
+    - cron:  '0 5 * * 0'
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.12' 
+      # - name: update requirements
+      #   run: |
+      #     pip install pip-tools
+      #     rm requirements.txt requirements/dev.txt
+      #     make requirements.txt requirements/dev.txt
+      - name: create project
+        run: |
+          pip install django
+          django-admin startproject --template="$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/zipball/$GITHUB_REF_NAME" --extension=py,rst,gitignore,cfg,in,yml,json,dockerignore --name=Makefile,Dockerfile testproj
+      - name: start services
+        run: |
+          docker compose up -d
+          # docker attach $(docker compose ps -q django-migrate)
+        working-directory: testproj
+     # ...
```

Comment:
Could you implement the rest of the tests from the description in #15? (curl and verify services are running)